lib/commands/build.ts

```diff
@@ -1,11 +1,35 @@
 ///<reference path="../.d.ts"/>
 "use strict";
 
-export class BuildAndroidCommand implements ICommand {
-	constructor(private $buildService: Project.IBuildService,
-		private $devicePlatformsConstants: Mobile.IDevicePlatformsConstants) { }
+import Future = require("fibers/future");
+import assert = require("assert");
 
+class BuildCommandBase implements ICommand {
+	constructor(private $project: Project.IProject,
+		private $errors: IErrors) { }
+	
 	allowedParameters: ICommandParameter[] = [];
+	
+	execute(args: string[]): IFuture<void> {
+		assert.fail("","", "You should never get here. Please contact Telerik support and send the output of your command, executed with `--log trace`.");
```

Comment:
yes, in case of future refactoring or new build command that extends BuildCommandBase, this assert will show the error.
We are using such asserts/errors.fail in other places as well :)